as-pect.config.js

```diff
@@ -13,6 +14,12 @@ export default {
   disclude: [/node_modules/],
   /**
    * Add your required AssemblyScript imports here.
+   * @param memorys - The WebAssembly.Memory instance.
+   * @param createImports - The createImports function.
+   * @param instantiate - The WebAssembly.instantiate function.
+   * @param binary - The WebAssembly binary.
+   * @returns The imports object.
```

Comment:
In case you're wondering what are the real types are : https://developer.mozilla.org/fr/docs/WebAssembly/JavaScript_interface/instantiate
The createImport from my understanding is simply a function that merge multiple objects.

Reply:
I don't understand what is the link between this instantiate method and our methods.

I didnt spend much time to find the sense behind this. Just update the code to fit with TypeDoc

Reply:
This is a configuration file, we don't need it in our generated library documentation, you can do the same as https://github.com/massalabs/massa-as-sdk/blob/main/as-pect.config.js